templates/footer.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Footer</title>
+    <link rel="stylesheet" href="/static/footer.css">
+  </head>
+
+  <body>
+    <div class="footer">
```

Comment:
You do not need the html core html tags in this code as it is included in to the structure that has it already. This file should start with this div and end where this div ends.

The stylesheet import can be done the way it was done in navigation.

Reply:
Well actually the file can end with the end of copyright div.